commercetools-sdk-base/src/main/java/io/sphere/sdk/client/SphereAuthConfig.java

```diff
@@ -37,6 +39,10 @@
         return DEFAULT_SCOPES;
     }
 
+    default List<String> getRawScopes() {
```

Comment:
Here we should add some javadoc to describe the difference between this method and the `getScopes()` method.